ensemble/ctf/utils.py

```diff
@@ -35,7 +38,11 @@ def save_ctf(transfer_func, filename):
     """
     function_data = transfer_func.to_dict()
     with open(filename, 'wb') as fp:
-        json.dump(function_data, fp, indent=1)
+        data = json.dumps(function_data, indent=1)
+        if six.PY2:
+            fp.write(data)
+        else:
+            fp.write(data.encode('utf-8'))
```

Comment:
`json.dumps` [always produces `str` objects](https://docs.python.org/3.6/library/json.html#json.dump)([py2](https://docs.python.org/2.7/library/json.html#json.dump) can produce `str` or `unicode`)? _And they dropped the `encoding` kwarg in Python 3?_

Really? 🙄